src/opf/utils.jl

```diff
@@ -72,7 +72,7 @@ function extract_metadata(opf::OPFModel{<:AbstractFormulation})
         "dual_status" => string(dual_status(model)),
         "solve_time" => solve_time(model),
         "primal_objective_value" => if has_values(model) objective_value(model) else Inf end,
-        "dual_objective_value" => if has_duals(model) dual_objective_value(model) else -Inf end,
+        "dual_objective_value" => try if has_duals(model) dual_objective_value(model) else -Inf end catch; -Inf end,
```

Comment:
@mtanneau I ran into some issue where with HiGHS, this `has_duals` would return true, but then trigger the error `Result index of attribute MathOptInterface.DualObjectiveValue(1) out of bounds. There are currently 0 solution(s) in the model.` when querying the dual objective value.

Reply:
Which test case / formulation was this?

Reply:
I think this may have been fixed upstream. I'd rather we remove the try-catch and open a separate issue if this problem re-surfaces down the line